Dnd/dndJavascript.js

```diff
@@ -1,7 +1,16 @@
-function showSeason(item) {
-    document.getElementById("seasonDropDownMenu").innerHTML = item.innerHTML;
-    clearEpisodes();
-    displaySeasonContent();
+  async function showSeason(item) {
+    if (item) {
+      document.getElementById("seasonDropDownMenu").innerHTML = item.innerHTML;
+      clearEpisodes();
+    }
+    const season_number = getSelectedSeasonNumber();
+    const episodeData = await getSeasonContent(season_number);
+    displaySeasonContent(episodeData);
+  }
+
+  function clearEpisodes() {
+    const episodes = document.querySelectorAll('[id=dnd-column]');
```

Comment:
IDs are meant to be unique, so why `querySelectorAll`?
Also, to find something with an ID, you should use `#dnd-column` and not `[id=dnd-column]`